src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2019,7 +2019,7 @@ s32 func_80834380(GlobalContext* globalCtx, Player* this, s32* itemPtr, s32* typ
         if (this->stateFlags1 & PLAYER_STATE1_23) {
             *typePtr = ARROW_NORMAL_HORSE;
         } else {
-            *typePtr = this->heldItemActionParam - 6;
+            *typePtr = this->heldItemActionParam - (PLAYER_AP_BOW - ARROW_NORMAL);
```

Comment:
Pretty minor but I think it would be easier to understand if it was written like this: (base value + offset)
```suggestion
            *typePtr = ARROW_NORMAL + this->heldItemActionParam - PLAYER_AP_BOW;
```

Reply:
I didn't know this kind of thing didn't break matching 👍